content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: With Gateways
+description: Istio mesh spanning multiple Kubernetes clusters using Istio Gateway to reach remote pods.
```

Comment:
The description is unclear. I suggest keeping the parallel construct:
Install an Istio mesh across multiple Kubernetes clusters using an Istio gateway